src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,39 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDate;
```

Comment:
Static imports should have precedence, followed by Java's imports, then followed by this Test import. Maybe when you run checkstyle, you would be clearer. At first, I did not know too! 😄 
```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;

import java.time.LocalDate;

import org.junit.jupiter.api.Test;
```